packages/nodes-base/nodes/Squarespace/Squarespace.node.ts

```diff
@@ -0,0 +1,85 @@
+import {
+	INodeProperties,
+	INodeType,
+	INodeTypeDescription,
+} from 'n8n-workflow';
+
+import { squarespaceApiPagination } from './GenericFunctions';
+import { inventoryFields, inventoryOperations } from './description/InventoryDescription';
+import { productFields, productOperations } from './description/ProductDescription';
+import { profileFields, profileOperations } from './description/ProfileDescription';
+import { transactionFields, transactionOperations } from './description/TransactionDescription';
+
+const resource: INodeProperties = {
+	displayName: 'Resource',
+	name: 'resource',
+	type: 'options',
+	noDataExpression: true,
+	options: [
+		{
+			name: 'Inventory',
+			value: 'inventory',
+		},
+		{
+			name: 'Product',
+			value: 'product',
+		},
+		{
+			name: 'Profile',
+			value: 'profile',
+		},
+		{
+			name: 'Transaction',
+			value: 'transaction',
+		},
+	],
+	default: '',
```

Comment:
This should trigger [`node-param-default-wrong-for-options`](https://github.com/ivov/eslint-plugin-n8n-nodes-base/blob/master/docs/rules/node-param-default-wrong-for-options.md). Does it not?

Reply:
Answering myself, it does not, will check.